### ISSUE TITLE
Named export too aggresively removed

### ISSUE THREAD

**Issue by dmail** [eight04/rollup-plugin-external-globals#11]
https://github.com/eight04/rollup-plugin-external-globals/blob/791f3f3eda6bdca61412ff75124def808ed2157b/lib/import-to-globals.js#L51-L54

Should ensure export source has a name no ?

```js
function analyzeExportNamed(node, code, names, tempNames) { 
   if (node.declaration || !node.source || !names.hasOwnProperty(node.source.value)) { 
     return false; 
   } 
```

**Comment by dmail** [comment]
❤️

**Comment by eight04** [comment]
Fixed in v0.3.1
